test/test.js

```diff
@@ -0,0 +1,33 @@
+const {getIPInfo} = require('../lib/ipinfo');
+const database = require('../lib/database');
+const TestUtil = require('./test-util');
+const assert = require('assert');
+
+const ipInfoTestData = TestUtil.getIPInfoTestData();
+
+describe('IP Info', function () {
```

Comment:
Could you use arrow function instead?

Reply:
Arrow functions shouldn't be used as they would change the context and 'this' would not be bind to Mocha.